src/VisualStudio/VisualBasic/Impl/LanguageService/VisualBasicCreateServicesOnTextViewConnection.vb

```diff
@@ -30,5 +32,16 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
 
             MyBase.New(workspace, globalOptions, listenerProvider, threadingContext, languageName:=LanguageNames.VisualBasic)
         End Sub
+
+        Protected Overrides Function InitializeServiceForOpenedDocumentAsync(document As Document) As Task
+            ' Preload project completion providers on a background thread since loading extensions can be slow
+            ' https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1488945
+            Dim compeltionServiceWithProviders = TryCast(document.GetRequiredLanguageService(Of CompletionService)(), CompletionServiceWithProviders)
+            If compeltionServiceWithProviders IsNot Nothing Then
+                compeltionServiceWithProviders.GetProjectCompletionProviders(document.Project)
+            End If
+
+            Return Task.CompletedTask
+        End Function
```

Comment:
this logic is the same as the block in teh c# type.  Move into AbstractCreateServicesOnTextViewConnection?